pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/AndOperatorTransformFunction.java

```diff
@@ -45,4 +45,13 @@ int getLogicalFuncResult(int left, int right) {
     }
     return 0;
   }
+
+  private boolean isFalse(int i) {
```

Comment:
why need this?

Reply:
Removed the function.

Had this function because another reviewer suggested that: https://github.com/apache/pinot/pull/11546#discussion_r1319452061.

Reply:
Ack, I don't think we need it here until one more reference is added.